Sources/Security/Signing.swift

```diff
@@ -276,6 +283,7 @@ extension Signing.SignatureParameters {
             nonce +
             path +
             postParameterHash +
+            headerParametersHash +
```

Comment:
Thanks to unit tests I was able to get this right. Initially I had this before the post hash.